### PR TITLE
chore: releases: merge releases branch to master

### PR DESCRIPTION
**Pull request by rjan90**
## Related Issues
Closes https://github.com/filecoin-project/lotus/issues/11476

## Proposed Changes

Last step of the release-flow for v1.25.2, merging the releases branch into master with conflicts resolved.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,202 @@
...
+- feat: mpool/wdpost: Maximize feecap config ([filecoin-project/lotus#9746](https://github.com/filecoin-project/lotus/pull/9746))
+   - Adds a Maximixe FeeCap Config
+- feat: Add lotus-bench cli option to stress test any binary ([filecoin-project/lotus#11270](https://github.com/filecoin-project/lotus/pull/11270))
+   - Enables the `Lotus-Bench` to run any binary and analyze their latency and histogram distribution, track most common errors, perform stress testing under different concurrency levels and see how it works under different QPS.
+- feat: chain import: don't walk to genesis - 2-3x faster snapshot import (#11446) ([filecoin-project/lotus#11446](https://github.com/filecoin-project/lotus/pull/11446))
+   - Improves Snapshot import speed, by not walking back to genesis on import.
 - feat: metric: export Mpool message count ([filecoin-project/lotus#11361](https://github.com/filecoin-project/lotus/pull/11361))
-- feat: sealing: load SectorsSummary from sealing SectorStats instead of calling API each time ([filecoin-project/lotus#11353](https://github.com/filecoin-project/lotus/pull/11353))
```
Comment (aarshkshah1992) [inline CHANGELOG.md:6]: why is this removed ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,21 +1,202 @@
...
+
+# v1.25.1 / 2023-12-09
+
+This is a **highly recommended PATCH RELEASE.**  The patch release fixes the issue were node operators trying to catch up sync were unable to sync large message blocks/epochs due to an increased number of messages on the network.
+
+This patch release allows for up to 10k messages per block. Additionally, it introduces a limit on the amount of data that can be read at once, ensuring the system can handle worst-case scenarios.
 
 ## Improvements
-- fix: Add time slicing to splitstore purging step during compaction to reduce lock congestion [filecoin-project/lotus#11269](https://github.com/filecoin-project/lotus/pull/11269)
-- feat: Added instructions on how to setup Prometheus/Grafana for monitoring a local Lotus node [filecoin-project/lotus#11276](https://github.com/filecoin-project/lotus/pull/11276)
```
Comment (aarshkshah1992) [inline CHANGELOG.md:12]: did you mean to remove all this ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,21 +1,202 @@
...
-- feat: Unambiguously translate native messages to Ethereum transactions by:
-  - Detecting native messages that "look" like Ethereum transactions (creating smart contracts, invoking a smart contract, etc.), and decoding them as such.
-  - Otherwise, ABI-encoding the inputs as if they were calls to a `handle_filecoin_method` Solidity method.
-- fix: ensure that the Ethereum API never returns "empty" addresses for native messages. When a "to" address cannot be resolved to a 0x-style address, it will be re-written to `0xff0000000000000000000000ffffffffffffffff`. This can only happen when the native transaction _reverted_ (failing to create an account at the specified "to" address).
+- fix: exchange: allow up to 10k messages per block ([filecoin-project/lotus#11506](https://github.com/filecoin-project/lotus/pull/11506))
+
+>>>>>>> releases
```
Comment (aarshkshah1992) [inline CHANGELOG.md:199]: is this from resolving a merge conflict ?

File: CHANGELOG.md (hunk 1 above)
Comment (rjan90) [reply CHANGELOG.md:6]: This has been moved to the improvements section under v1.25.2: https://github.com/filecoin-project/lotus/blob/330afff2e917d23c1ef57bf5681508b695f7364d/CHANGELOG.md?plain=1#L75.

All the PRs that was previously in the **Unreleased** section has been removed from this section, and been moved to somewhere in the `v1.25.2` changelog, as they are now no longer unreleased.

File: CHANGELOG.md (hunk 2 above)
Comment (rjan90) [reply CHANGELOG.md:12]: Yes, these has now been moved to the v1.25.2 changelog, as these PRs are not `Unreleased` anymore. You can see these PRs here:
https://github.com/filecoin-project/lotus/blob/330afff2e917d23c1ef57bf5681508b695f7364d/CHANGELOG.md?plain=1#L72C1-L74

File: CHANGELOG.md (hunk 3 above)
Comment (rjan90) [reply CHANGELOG.md]: Yeah, that looks like an artefact from a merge conflict. Pushed a fix to remove it here: https://github.com/filecoin-project/lotus/pull/11578/commits/73d830e2fb89c4b0f077771b465cf959dfd2d105

**Review by arajasek — APPROVED** [review]
